libs/common-services/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -61,6 +61,20 @@ public class ConfigService {
     private final SecretsProvider secretsProvider;
     private final ObjectMapper objectMapper = new ObjectMapper();
 
+    public enum featureFlags {
+        RESET_IDENTITY("resetIdentity");
```

Comment:
We have an enum already for feature flags - in `library/config/CoreFeatureFlag.java` so I think we can just add this as a value there?